tests/fixtures/integration/actions/templar/test_welcome_interactive_noee.py/test/8.json

```diff
@@ -33,7 +33,7 @@
         " 19│      - This action will not work normally outside of lockstep strategies",
         " 20│      support: none",
         " 21│    check_mode:",
-        " 22│      description: Can run in check_mode and return changed status prediction withought",
+        " 22│      description: Can run in check_mode and return changed status prediction without",
```

Comment:
this one can stay, because I recall us adding an exception for the fixtures folder, and it will get updated later when the fixtures are updated